CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+3.0.0 (unreleased)
```

Comment:
If we don't know the date, let's say `in progress` instead of `unreleased`.

Reply:
Updated, thanks!